### PR TITLE
Imports TG Pills - Matches pill bottle aesthetic

### PR DESCRIPTION
**Pull request by ItsVyzo**
# About the pull request

Imports TG Pills - Matches pill bottle aesthetic

# Explain why it's good for the game

consistency + nicer sprites

# Testing Photographs and Procedure

<details>

<summary>Screenshots & Videos</summary>

pills
pills2
pills3
pills4
pills5
pills6
pills7
pills8
pills9
pills10
pills11
pills12
pills13
pills14
pills15
pills16
pills17
pills18
pills19
pills20
pills21
pills22

</details>

# Changelog

:cl: LTNTS
imageadd: port tgsprites for pills (including a new variation)
/:cl:

### CODE REVIEW

**Review by Blundir — APPROVED** [review]
sprites look good

**Comment by Blundir** [comment]
Also document that some new pills are not in selection

File: code/modules/reagents/chemistry_machinery/chem_master.dm [hunk 1]
```diff
@@ -291,7 +291,7 @@
 				if(!Adjacent(usr) || !usr.put_in_hands(P))
 					P.forceMove(loc)
 		else if(href_list["change_pill"])
-			#define MAX_PILL_SPRITE 20 //max icon state of the pill sprites
+			#define MAX_PILL_SPRITE 22 //max icon state of the pill sprites
```
Comment (Drulikar) [inline code/modules/reagents/chemistry_machinery/chem_master.dm:294]:
```suggestion
```

File: code/modules/reagents/chemistry_machinery/chem_master.dm [hunk 2]
```diff
@@ -291,7 +291,7 @@
...
 			var/dat = "<table>"
 			for(var/i = 1 to MAX_PILL_SPRITE)
```
Comment (Drulikar) [inline code/modules/reagents/chemistry_machinery/chem_master.dm:296]:
```suggestion
			for(var/i = 1 to PILL_ICON_CHOICES)
```

File: code/__DEFINES/chemistry.dm [hunk 3]
```diff
@@ -3,7 +3,7 @@
  */
 
 /// Amount of random icon variations for pills in total
```
Comment (Drulikar) [inline code/__DEFINES/chemistry.dm:5]:
```suggestion
/// Amount of bottle icon variations in total
#define BOTTLE_ICON_CHOICES 4
/// Amount of random icon variations for pills in total
```

File: code/modules/reagents/chemistry_machinery/chem_master.dm
Comment (Drulikar) [inline code/modules/reagents/chemistry_machinery/chem_master.dm:1]: - Remove the line `#define MAX_BOTTLE_SPRITE 4 //max icon state of the bottle sprites`
- Rename the other usage of `MAX_PILL_SPRITE` in this file to `PILL_ICON_CHOICES`
- Rename the usages of `MAX_BOTTLE_SPRITE` to `BOTTLE_ICON_CHOICES`

**Review by Drulikar — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry_machinery/chem_master.dm
Comment (ItsVyzo) [reply code/modules/reagents/chemistry_machinery/chem_master.dm:1]: All done

Comment (ItsVyzo) [reply code/modules/reagents/chemistry_machinery/chem_master.dm:1]: I think

Comment (Drulikar) [reply code/modules/reagents/chemistry_machinery/chem_master.dm:1]: The mentions here I cannot suggest because they are not within review context to suggest changes to.

File: code/__DEFINES/chemistry.dm [hunk 4]
```diff
@@ -2,8 +2,6 @@
  * Chemistry defines
  */
 
-/// Amount of bottle icon variations in total
-#define BOTTLE_ICON_CHOICES 4
```
Comment (Drulikar) [inline code/__DEFINES/chemistry.dm]: Why did you remove what I just suggested to be added

Comment (ItsVyzo) [reply code/__DEFINES/chemistry.dm]: I think fixed

File: code/modules/reagents/chemistry_machinery/chem_master.dm
Comment (ItsVyzo) [reply code/modules/reagents/chemistry_machinery/chem_master.dm:1]: yeah I screwed up. I think I fixed it now qq

**Review by Drulikar — APPROVED** [review]
